File: sources/com/lumiyaviewer/lumiya/slproto/users/manager/_$Lambda$x6PlkRNg0xExeA_EUn8oEJWcOq8.java
/*
 * Decompiled with CFR 0.152.
 */
package com.lumiyaviewer.lumiya.slproto.users.manager;

import com.lumiyaviewer.lumiya.react.Subscription;
import com.lumiyaviewer.lumiya.slproto.users.manager.ChatterSubscription;
import com.lumiyaviewer.lumiya.voice.common.model.VoiceChatInfo;

/*
 * Illegal identifiers - consider using --renameillegalidents true
 */
final class _$Lambda$x6PlkRNg0xExeA_EUn8oEJWcOq8
implements Subscription.OnData {
    private final Object -$f0;

    private final /* synthetic */ void $m$0(Object object) {
        ((ChatterSubscription)this.-$f0).-com_lumiyaviewer_lumiya_slproto_users_manager_ChatterSubscription-mthref-0((VoiceChatInfo)object);
    }

    public /* synthetic */ _$Lambda$x6PlkRNg0xExeA_EUn8oEJWcOq8(Object object) {
        this.-$f0 = object;
    }

    public final void onData(Object object) {
        this.$m$0(object);
    }
}

File: Linkpoint/src/main/app_broken_backup/com/linkpoint/slproto/users/manager/$Lambda$x6PlkRNg0xExeA_EUn8oEJWcOq8.kt
package com.linkpoint.slproto.users.manager

import com.linkpoint.react.Subscription
import com.linkpoint.voice.common.model.VoiceChatInfo

/* renamed from: com.linkpoint.slproto.users.manager.-$Lambda$x6PlkRNg0xExeA_EUn8oEJWcOq8  reason: invalid class name */
/* synthetic */ class $Lambda$x6PlkRNg0xExeA_EUn8oEJWcOq8 : Subscription.OnData {

    /* renamed from: -$f0  reason: not valid java name */
    private /* synthetic */ Any f222$f0

    private /* synthetic */ Unit $m$0(Any obj) {
        ((this as ChatterSubscription).f222$f0).m297com_lumiyaviewer_lumiya_slproto_users_manager_ChatterSubscriptionmthref0((VoiceChatInfo) obj)
    }

    /* synthetic */ $Lambda$x6PlkRNg0xExeA_EUn8oEJWcOq8(Any obj) {
        this.f222$f0 = obj
    }

    fun onData(Any obj): Unit {
        $m$0(obj)
    }
}
